.travis.yml

```diff
@@ -33,18 +33,22 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.7.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.*@dev
+      env: SYMFONY_VERSION=2.8.*
     - php: 5.6
-      env: SYMFONY_VERSION="3.0.x-dev" DEPENDENCIES="dev" COMPOSER_FLAGS="--prefer-stable"
+      env: SYMFONY_VERSION=3.0.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.1.*@dev
   allow_failures:
     - php: 7.0
     - php: hhvm
+    - env: SYMFONY_VERSION=3.1.*@dev
 
 before_script:
   - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer selfupdate
   - composer config -q github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - if [ "$SYMFONY_VERSION" = "2.8.*@dev" ] || [ "$SYMFONY_VERSION" = "3.0.x-dev" ]; then SYMFONY_DEPRECATIONS_HELPER=strict; fi;
+  - if [ "$SYMFONY_VERSION" = "2.8.*" ] || [ "$SYMFONY_VERSION" = "3.0.*" ]; then SYMFONY_DEPRECATIONS_HELPER=strict; fi;
```

Comment:
Didn't see this but why did you keep this line? Strict deprecation is set only for allowed failure.

Please remove it.

Note: I think this could be entirely removed when all deprecation will be fixed, on another PR.